build-tools/installers/create-installers.targets

```diff
@@ -159,6 +159,7 @@
     <_MSBuildFiles Include="$(MSBuildSrcDir)\LayoutBinding.cs" />
     <_MSBuildFiles Include="@(AndroidSupportedTargetJitAbi->'$(MSBuildSrcDir)\lib\%(Identity)\libmono-android.debug.so')"       Condition=" '$(PackageId)' != 'Microsoft.Android.Sdk' " />
     <_MSBuildFiles Include="@(AndroidSupportedTargetJitAbi->'$(MSBuildSrcDir)\lib\%(Identity)\libmono-android.release.so')"     Condition=" '$(PackageId)' != 'Microsoft.Android.Sdk' " />
+    <_MSBuildFiles Include="@(AndroidSupportedTargetJitAbi->'$(MSBuildSrcDir)\lib\%(Identity)\libxa-internal-api.so')"          Condition=" '$(PackageId)' != 'Microsoft.Android.Sdk' " />
```

Comment:
[Elsewhere](https://xamarinhq.slack.com/archives/C03CEGRUW/p1595283571077500), @garuma asked:

> I tried installing the .vsix from your PR and I see some libxa-internal-api.so but nothing similar for desktop


We need to update the installer to likewise install `libxa-internal-api.dylib` (macOS) and `libxa-internal-api.dll` (Windows).  (Related question: shouldn't it be `xa-internal-api.dll` on Windows, *without* the `lib` prefix?)

These files *are* built, as per the log file, they're just not in the installers.

Reply:
Fixed the installers. Regarding the name, I considered it but all the other files have the `lib` prefix so I decided to keep it - it's just a convention and the Designer loads them directly with the `lib` prefix already, so I don't think there's a point in changing name of this new DLL.